### DISCUSSION TITLE
how to hide the icon in the window header

### DISCUSSION THREAD

**Discussion by iiYeung** [YiiGuxing/TranslationPlugin#1866] [Q&A]
how to hide the icon in the window

**Comment by YiiGuxing** [comment]
Which icon are you referring to?

**Reply by iiYeung** [reply]
![image](https://user-images.githubusercontent.com/19946752/183854799-275ea38c-f630-4786-9d49-aa7a5908dfdf.png)

**Reply by YiiGuxing** [reply]
It can't be hidden yet.
